index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
Please add a [`<link>`](https://www.w3schools.com/tags/tag_link.asp) tag here to add Css stylesheet to the page like below :point_down: 
`<link rel="stylesheet" href="styles.css">`